features/app/components/AbstractApp.js

```diff
@@ -22,6 +31,30 @@ export class AbstractApp extends Component {
     }
 
     /**
+     * Init lib-jitsi-meet and create local participant when component is going
+     * to be mounted.
+     *
+     * @inheritdoc
+     */
+    componentWillMount() {
+        let dispatch = this.props.store.dispatch;
+        dispatch(localParticipantJoined());
+        dispatch(initLib(this.props.config));
```

Comment:
I'm sorry, I may have asked the following question in person but I'm not sure so I'd rather have it & the answer in writing:

Why is initLib dispatched after localParticipantJoined & not in response to localParticipantJoined?